src/app/cube/details/details.component.html

```diff
@@ -37,8 +37,15 @@
           (deleteRating)="deleteRating($event)" (editResponse)="editResponse($event);" (deleteResponse)="deleteResponse($event)" (reportRating)="reportRating($event)" (respondRating)="submitResponse($event)"></clark-learning-object-ratings>
       </div>
     </div>
-    <div class="sidebar">
-      <cube-details-action-panel [learningObject]="learningObject"></cube-details-action-panel>
+    <div class="sidebar" [ngStyle] = "{'margin-top': reviewer ? '-190px' : '-110px'}">
+      <div class="inner"*ngIf="reviewer">
+        <div class="revisions">
```

Comment:
Couldn't this be just one *ngIf="hasRevisions" statement instead of two?  Maybe an else statement too.